auth-service/src/main/kotlin/uk/gov/communities/delta/auth/utils/ADGroup.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.communities.delta.auth.utils
+
+import uk.gov.communities.delta.auth.config.LDAPConfig
+import javax.naming.directory.Attribute
+import javax.naming.directory.BasicAttribute
+
+class ADGroup(val cn: String, private val ldapConfig: LDAPConfig){
```

Comment:
Same for ADUser, I think I'd make this an inner class of one of the services, it feels like an internal helper class and not something we should be returning back to controllers or other parts of the code. If it's here other code might start using it that shouldn't,